client/transactions/list/index.js

```diff
@@ -41,14 +41,12 @@ import TransactionsFilters from '../filters';
 import Page from '../../components/page';
 import wcpayTracks from 'tracks';
 
-const getColumns = ( includeDeposit, includeSubscription, sortByDate ) =>
+export const getColumns = ( includeDeposit, includeSubscription ) =>
 	[
 		{
-			key: 'details',
-			label: '',
-			required: true,
-			// Match background of details and date when sorting.
-			cellClassName: 'info-button ' + ( sortByDate ? 'is-sorted' : '' ),
+			key: 'transaction_id' /* eslint-disable camelcase */,
```

Comment:
This comment disables linter rule till matching enable comment (or end of the file in this case). Moreover, since `_` is used in value linter should be happy even without that comment :)

Reply:
Ah! my bad, earlier I was using snake_case as the key and forgot to remove it, fixed it in commit https://github.com/Automattic/woocommerce-payments/pull/1776/commits/10f4a6afe92276ea1c89fd55758b5ba90af50358